platform/android/app/src/main/kotlin/coop/polypoly/polypod/PodApiError.kt

```diff
@@ -2,4 +2,6 @@ package coop.polypoly.polypod
 
 class PodApiError(message: String = "") : Exception(message) {
     fun endpointError() = PodApiError("unable to contact endpoint")
+    fun endpointPermissionDenied() =
+        PodApiError("user denied request to endpoint")
```

Comment:
You'll probably want to clarify this. Is this specific user denied the request? Or is it a generic issue? Is it because the user lacks permissions?

Reply:
added this for more clarification actually. not that happy with all the api errors we've currently but avoided touching it more here. this is used only when user 'denies access to endpoint' - used on error reports here. so we can tell apart the different errors